.github/workflows/github-merit-badger.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: github-merit-badger
+on:
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  call-action:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: aws-github-ops/github-merit-badger@main
+        id: merit-badger
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          badges: '[beginning-contributor,repeat-contributor,valued-contributor,admired-contributor,star-contributor,distinguished-contributor]'
+          thresholds: '[0,3,6,13,25,50]'
```

Comment:
Just a recommendation (ignore if you don't see fit): 
`beginning-contributor` -> `first-time-contributor` or `beginner-contributor`
`valued` -> `seasoned`?

Reply:
@bbarani any thoughts?

Reply:
My recommendation ..... first-time-contributor -> repeat-contributor -> valued-contributor -> seasoned-contributor -> all-star-contributor -> distinguished-contributor

Reply:
@bbarani updated the workflow
```
badges: '[first-time-contributor,repeat-contributor,valued-contributor,seasoned-contributor,all-star-contributor,distinguished-contributor]'
```